extension/src/content_scripts/simplification.js

```diff
@@ -24,9 +24,10 @@
             main: "main, [role~='main'], .main, #main",
             genericContent: ".content, #content, .body:not('body'), #body:not('body')",
             nav: "nav, [role~='navigation'], .navigation, .nav, #nav, #navigation",
-            search: "[role~='search'], [type~='search']"
+            search: "[role~='search'], [type~='search']",
```

Comment:
This whole component could deal with a good deal of docs explaining its purpose, strategy, what kind of contract it conforms to, etc. (right now it just appears to be a viewComponent - does it comply with any other requirements?)

Reply:
Methods, especially, such as "findContent" should have reasonable JSDocs, etc.